test/index.js

```diff
@@ -79,6 +79,19 @@ describe('Metal Tools - Soy', function() {
         done();
       });
     });
+
+    it('should use custom handleError function', function(done) {
+      var stream = metalToolsSoy({
+        dest: 'test/fixtures/soy',
+        handleError: function(err) {
+          assert.ok(err);
+          assert.ok(err.message);
+
+          done();
+        },
+        src: 'test/fixtures/soy/compileError.soy'
+      });
```

Comment:
Hey @Robert-Frampton, just a little nitpicking... asserting and existing through the `handleError` seems weird... shouldn't we do something like this instead?

```js
var handleError = sinon.spy();

[...] 

stream.on('end', function() {
    assert.strictEqual(1, handleError);
    done();
});
```